dkist/tests/test_benchmarks.py

```diff
@@ -29,6 +34,7 @@ def test_pixel_to_world(benchmark, visp_dataset_no_headers):
 
 
 @pytest.mark.benchmark
+@pytest.mark.walltime
```

Comment:
I assume then that `pytest.mark.benchmark` and `pytest.mark.walltime` are not mutually exclusive?